pom.xml

```diff
@@ -378,6 +378,7 @@
                         <systemPropertyVariables>
                            <!-- See test org.hibernate.ogm.test.utils.EnvironmentTest -->
                            <hibernate.service.allow_crawling>false</hibernate.service.allow_crawling>
+                           <user.timezone>UTC</user.timezone>
```

Comment:
Error is that: Test stores time in UTC to database. But database stores the time with JDK timezone. And test read from database incorrect time.  You not see the problem because timezone of your JDK and timezone in test is SAME. I see problem because test's timezone is UTC+0, BUT JDK's timezone is UTC+3!!!